src/features/accounts/components/accounts-menu/add-account-modal.tsx

```diff
@@ -172,18 +173,21 @@ function CreateAccountOptions({
 }: {
   onAddMethodClick: (method: AddAccountMethodTypes) => void;
 }) {
+  const { services } = useNeighborhoodContext();
```

Comment:
Now that services is part of the context, I just pull it out at the component level instead of setting it in the container and passing it down through props.